src/quo/components/profile/collectible_list_item/style.cljs

```diff
@@ -0,0 +1,104 @@
+(ns quo.components.profile.collectible-list-item.style
+  (:require [quo.foundations.colors :as colors]
+            [quo.foundations.shadows :as shadows]))
+
+(def type-card-image-height 150)
+(def type-card-image-height-and-border (+ type-card-image-height 2))
```

Comment:
Should this use the `borders-width` defined below? `(def type-card-image-height-and-border (+ type-card-image-height borders-width))`

Reply:
will double check this!

Reply:
thanks, so in the end I actually removed these as not needed since I made this responsive 👍